sample_files/.travis.yml

```diff
@@ -54,14 +54,14 @@ env:
   matrix:
   - LINT_CHECK="1"
   - TRANSIFEX="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
+  - TESTS="1" ODOO_REPO="odoo/odoo" MQT_TEMPLATE_DB='mqt_odoo_template_core' MQT_TEST_DB='mqt_odoo_core'
+  - TESTS="1" ODOO_REPO="OCA/OCB" MQT_TEMPLATE_DB='mqt_odoo_template_ocb' MQT_TESTE_DB='mqt_odoo_ocb'
 # either use the two lines above or the two below. Don't change the default if
 # it's not necessary (it is only necessary if modules in your repository can't
 # be installed in the same database. And you get a huge speed penalty in your
 # tests)
-#  - TESTS="1.0" ODOO_REPO="odoo/odoo" UNIT_TEST="1"
-#  - TESTS="1.0" ODOO_REPO="OCA/OCB" UNIT_TEST="1"
+#  - TESTS="1.0" ODOO_REPO="odoo/odoo" UNIT_TEST="1" MQT_TEMPLATE_DB='mqt_odoo_template_core' MQT_TEST_DB='mqt_odoo_core'
+#  - TESTS="1.0" ODOO_REPO="OCA/OCB" UNIT_TEST="1" MQT_TEMPLATE_DB='mqt_odoo_template_ocb' MQT_TESTE_DB='mqt_odoo_ocb'
```

Comment:
These variables are not required, so I would rather not include them in the quickstart travis.yml file.
On the other hand, it would be good to have these options documented on the README.